### ISSUE TITLE
Wildcard entity (Dialogflow sys.any functionality)

### ISSUE THREAD

**Issue by dcalvom** [samtecspg/articulate#192]
### Feature Request

@BreadAndRoses95 mention on gitter he would like to be able to add a wildcard entity that would match any non-empty input.

Let's say you are looking for files and you don't know all the possible filenames so you would tag that text with this new entity and that will help you to recognize whatever input the user used.

**Comment by wrathagom** [comment]
So the CRF should already be able to generalize, especially on filenames, I imagine it would work quite well as I would expect them to have slashes or dots in them, which make them fairly unique.

His use case of some sort of order number with `INC` at the beginning is a great use case for the regex patterns in #39 

This is still related to #193 in a way that we would like a way to train the responses better.

But @dcalvom how did you envision this, is it literally a way indicate that whatever follows should be the answer to the prompt? What happens if the user changes intents mid slow-filling and goes from ordering pizza to ordering airplane tickets?

**Comment by BreadAndRoses95** [comment]
People may have to clearly validate the slot filling if the wildcard is used, but I do think that this is a great feature to propose (let say we want the user to describe the content of the ticket, making an entity that is ambigious enough to match any kind of description is complicated/impossible, so this wildcard would be nice). I asked for PR for the regex feature, let me know if it suits you or not since I'm not fully aware of the full architecture of the project.

**Comment by wrathagom** [comment]
@BreadAndRoses95 okay, so the example that I have in my head is a customer trying to open a support ticket.

**Customer:**
I'd like to report an issue. <- get's recognized as createTicket intent and slot filling begins.
**Bot:**
I can help you with that, can you please describe your issue? <- prompt from wildcard slot
**Customer:**
Well two days ago my computer started smoking a little bit, but it wasn't like on fire or anything so I just ignored it. But now today it won't turn on. <- wildcard response, this entire text will be added to slot value.
**Bot:** Okay, got it. We've created a support ticket, someone should contact you shortly.

We're just accepting the fact that if the customer replies with "Actually I want to order a pizza" that we're going end up filling the slot with that value?

**Comment by wrathagom** [comment]
Hopefully no one has started on this. I am close to an implementation proposal as part of a larger enhancement. So this one is waiting on me.

**Comment by wrathagom** [comment]
Closing in favor of #900
